cmd/utils/flags.go

```diff
@@ -751,6 +757,10 @@ func RegisterEthService(ctx *cli.Context, stack *node.Node, extra []byte) {
 				peerIds := make([]string, len(peers))
 
 				for peerIdx, peer := range peers {
+					if peer.RaftPort == 0 {
```

Comment:
An example might be worthwhile. I don't think I would know what to do if I saw this.